v2/elasticsearch-common/src/main/java/com/google/cloud/teleport/v2/elasticsearch/options/ElasticsearchWriteOptions.java

```diff
@@ -0,0 +1,93 @@
+package com.google.cloud.teleport.v2.elasticsearch.options;
```

Comment:
Please move to `com.google.cloud.teleport.v2.options` package (no "elasticsearch" in the package name).

Reply:
@an2x -  To provide context, since these options are defined in elasticsearch-common, our earlier suggestion is to include elasticsearch in pacakage name (i.e com.google.cloud.teleport.v2.elasticsearch) prior to all options, transforms and utils child packages.  We are following similar pattern with other modules (i.e kafka, snowflake etc) as well.

if you are fine, probably we can delete other comments to avoid confusion.

Reply:
Makes sense, thanks.